app/views/tasks/_task.html.haml

```diff
@@ -0,0 +1,14 @@
+
+.row
+  .card.col-sm-12
+    .row.card-body{id: "card#{@task.id}"}
```

Comment:
update all html ids to be of format `element-#{rubyObject.id}`

Reply:
not working